commons/push/integration/garmin/garmin_push_activity_details.avsc

```diff
@@ -5,42 +5,33 @@
-        {"name": "startingLongitudeInDegree", "type": ["null", "double"], "doc": "Starting Longitude in Degree", "default": null},
-        {"name": "steps", "type": ["null", "int"], "doc": "Count of steps recorded during the monitoring period.", "default": null},
-        {"name": "totalElevationGainInMeters", "type": ["null", "double"], "doc": "Total Elevation Gain in meters", "default": null},
-        {"name": "totalElevationLossInMeters", "type": ["null", "double"], "doc": "Total Elevation Loss in meters", "default": null},
-        {"name": "isParent", "type": ["null", "boolean"], "doc": "If present and set to true, this activity is the parent activity of one or more child activities that should also be made available in the data feed to the partner. An activity of type MULTI_SPORT is an example of a parent activity.", "default": null},
-        {"name": "parentSummaryId", "type": ["null", "string"], "doc": "If present, this is the summaryId of the related parent activity. An activity of type CYCLING with a parent activity of type MULTI_SPORT is an example of this type of relationship.", "default": null},
-        {"name": "manual", "type": ["null", "boolean"], "doc": "Indicates that the activity was manually entered directly on the Connect site. This property will only exist for manual activities.", "default": null},
-        {"name": "deviceName", "type": ["null", "string"], "doc": "Only Fitness Activities are associated with a specific Garmin device rather than the user’s overall account. If a user wears two devices at once during the same time and starts a Fitness Activity on each then both will generate separate Activity summaries with two different deviceNames. Always ‘unknown’ for manually created activities.", "default": null}
-      ]
-      }
-    },
-    {"name": "samples", "type": { "type": "array", "items" : "GarminActivityDetailsSample", "default": [] }, "doc": "Each activity detail contains an optional list of samples. The samples list will be empty if the activity is manual or details are not supported by the device. Samples may be as frequent as once per second, and values should be considered valid until the next sample."}
-  ]
+    {"name": "time", "type": "double", "doc": "Start time of the activity in seconds since January 1, 1970, 00:00:00 UTC (Unix timestamp)."},
+    {"name": "timeReceived", "type": "double", "doc": "Time that this record was collected by a service in seconds since the Unix Epoch (s)."},
+    {"name": "startTimeOffsetInSeconds", "type": ["null", "int"], "doc": "Offset in seconds to add to time to derive the 'local' time of the device that captured the data.", "default": null},
+    {"name": "activityType", "type": ["null", "string"], "doc": "Text description of the activity type. See Appendix A for a complete list.", "default": null},
+    {"name": "durationInSeconds", "type": ["null", "int"], "doc": "Length of the monitoring period in seconds.", "default": null},
```

Comment:
For these fields and the next, I’d opt for moving the unit out of the field name and just have it in the doc, like in the schema’s of other integrations.